ci/test/03_test_script.sh

```diff
@@ -33,7 +33,9 @@ echo "=== BEGIN env ==="
 env
 echo "=== END env ==="
 
-(
+if [ "$RUN_TIDY" != "true" ]; then
+  # The tidy task has no UB detector, so skip the patch there to avoid issues
```

Comment:
> to avoid issues

What issues? Do I understand correctly, that if we land the patch in the subtree, this will just break again?

Reply:
The following `git diff` prints this patch even IWYU did not apply any modifications.

Reply:
> Doesn't matter much, but the comment could also say "... so skip the patch because it is not needed and to avoid issues ...".

Thanks! [Adjusted](https://github.com/bitcoin/bitcoin/pull/31308#issuecomment-2936303614).